gcp_pilot/pubsub.py

```diff
@@ -179,12 +179,14 @@ async def create_subscription(
                 **(self.get_oidc_token(audience=push_to_url) if use_oidc_auth else {}),
             )
 
+        subscription = Subscription(
```

Comment:
This can be reused, and there's not standalone argument for `enable_message_ordering` (see next commit)